src/main.cpp

```diff
@@ -111,6 +111,11 @@ int main( int argc, char* argv[] ) {
     ///////////////////////////////
     // Are we restarting?
     if(input.restartRequested) {
+      if(input.forceInitRequested) {
+        idfx::pushRegion("Setup::Initflow");
+        mysetup.InitFlow(data);
+        idfx::popRegion();
+      }
```

Comment:
Should also call `DeriveVectorPotential()` otherwise this won't be compatible with VECTOR_POTENTIAL

Reply:
done